LICENSE

```diff
@@ -1,5 +1,7 @@
 Copyright 2023 teonite ventures sp. z o.o. (teonite)
 
+Note: The following license applies to the entire repository except for the "enterprise" directory.
```

Comment:
NOTE: The following license applies to the entire repository except for all the contents in the "enterprise" directory in this repository.
NOTE: For the license of the enterprise directory, go to enterprise/LICENSE.